entry.go

```diff
@@ -260,24 +260,48 @@ func (entry *Entry) Warningln(args ...interface{}) {
 }
 
 func (entry *Entry) Errorln(args ...interface{}) {
-	if entry.Logger.level() >= ErrorLevel {
+	if entry.IsErrorEnabled() {
 		entry.Error(entry.sprintlnn(args...))
 	}
 }
 
 func (entry *Entry) Fatalln(args ...interface{}) {
-	if entry.Logger.level() >= FatalLevel {
+	if entry.IsFatalEnabled() {
 		entry.Fatal(entry.sprintlnn(args...))
 	}
 	Exit(1)
 }
 
 func (entry *Entry) Panicln(args ...interface{}) {
-	if entry.Logger.level() >= PanicLevel {
+	if entry.IsPanicEnabled() {
 		entry.Panic(entry.sprintlnn(args...))
 	}
 }
 
+func (entry *Entry) IsDebugEnabled() bool {
```

Comment:
I don't think we need to implements these methods at Entry level. This property is more related to the logger and the forward call is simple enough to not add them to the Entry interface.

Reply:
Removed from `Entry`